torch/csrc/jit/codegen/cuda/executor_kernel_arg.h

```diff
@@ -95,6 +95,15 @@ struct DoubleArg : public ArgAbstract {
   }
 };
 
+struct ComplexDoubleArg : public ArgAbstract {
+  c10::complex<double> val_;
+  explicit ComplexDoubleArg(c10::complex<double> _val) : val_(_val) {}
+  // NOLINTNEXTLINE(modernize-use-override,cppcoreguidelines-explicit-virtual-functions)
```

Comment:
NITPICK: Totally not your problem. but it's a little strange that we went with nolint instead of putting a virtual + override in the function :smile: